### DISCUSSION TITLE
printer settings in portable version?

### DISCUSSION THREAD

**Discussion by cmoegele** [OrcaSlicer/OrcaSlicer#5825] [Q&A] [answered]
Is there a possibility to store the printer settings inside the orcaslicer folder? My idea of using the portable version was to have an actual version on the Nas folder and start it on different PCs or laptop by link. 

Thx for help in advance, best regards, cmoegele

**Comment by Avpman2** [comment]
This seems like a no-brainer. I'm not sure where the settings for the portable version are stored but they should NOT interfere with or otherwise alter an installed version's settings. IMHO

Thanks for all the hard work!

**Comment by HAndreasH** [comment, marked as answer]
maybe you find the answere here:
(https://github.com/SoftFever/OrcaSlicer/issues/1027#issuecomment-1542496692)
